src/pretix/control/forms/widgets.py

```diff
@@ -77,3 +77,18 @@ def options(self, name, value, attrs=None):
 
 class Select2ItemVarQuota(Select2ItemVarQuotaMixin, forms.Select):
     pass
+
+
+class Select2ItemVarQuotaMulti(Select2Mixin, forms.SelectMultiple):
+    def options(self, name, value, attrs=None):
+        for i, v in enumerate(value):
```

Comment:
After reading this function, I have still no idea what it does and why we need it. Could you add a short comment explaining its necessity?

Reply:
I added an inline comment as well now, but to explain: we need to have this to get multi-selection while having no queryset to use as we combine items and variations into the choice list (if we had, we could use the Select2 without change). We tried adjusting the Select2ItemVarQuotaMixin but it broke so much, so this is the cleanest way I myself can see.